### ISSUE TITLE
ImpermanentGraphDatabase uses filessystem

### ISSUE THREAD

**Issue by tbaum** [neo4j/neo4j#1873]
Using the `ImpermanentGraphDatabase` will create a subdirectory for the schema-index, shouldn't it stay in memory only?

````
# find target/test-data/
target/test-data//impermanent-db
target/test-data//impermanent-db/schema
target/test-data//impermanent-db/schema/index
target/test-data//impermanent-db/schema/index/lucene
target/test-data//impermanent-db/schema/index/lucene/1
target/test-data//impermanent-db/schema/index/lucene/1/failure-message
target/test-data//impermanent-db/schema/index/lucene/3
target/test-data//impermanent-db/schema/index/lucene/3/failure-message
target/test-data//impermanent-db/schema/index/lucene/4
target/test-data//impermanent-db/schema/index/lucene/4/failure-message
target/test-data//impermanent-db/schema/index/lucene/5
target/test-data//impermanent-db/schema/index/lucene/5/failure-message
....
`````

**Comment by tbaum** [comment]
@tinwelint this issue seems to be fixed in 2.3?

**Comment by tinwelint** [comment]
Yes, looks that way. Closing!
